### PR TITLE
BaSM add s3 metrics bucket to Production

### PR DESCRIPTION
**Pull request by martyn-w**
Adds s3 metrics bucket to book a secure move production environment

### CODE REVIEW

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-production/resources/s3-metrics.tf [hunk 1]
```diff
@@ -0,0 +1,98 @@
+/*
+ Based on https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/tree/main/example
+ */
+module "book_a_secure_move_metrics_s3_bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket?ref=4.5"
+
+  team_name              = var.team_name
+  business-unit          = "Digital and Technology"
```
Comment (digitalronin) [inline namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-production/resources/s3-metrics.tf:8]: The value of `business-unit` must be one of the values listed in the guidance here:
https://ministryofjustice.github.io/technical-guidance/documentation/standards/documenting-infrastructure-owners.html#mandatory
In this case, I think `HMPPS` is probably the one you want.

**Review by digitalronin — CHANGES_REQUESTED** [review]
Looks good. Just one small change to the business-unit value, please.

File: namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-production/resources/s3-metrics.tf (hunk 1 above)
Comment (martyn-w) [reply namespaces/live-1.cloud-platform.service.justice.gov.uk/hmpps-book-secure-move-api-production/resources/s3-metrics.tf]: Thanks David, now updated

**Review by digitalronin — APPROVED** [review]

**Comment by martyn-w** [comment]
Closing and re-opening to re-trigger github hooks - looks like something is stuck

**Pull request merged**